tests/system/provisioning/four_manager_disconnected_node/roles/agent-role/files/ossec.conf

```diff
@@ -0,0 +1,170 @@
+<!--
+  Wazuh - Agent - Default configuration for debian 10
+  More info at: https://documentation.wazuh.com
+  Mailing list: https://groups.google.com/forum/#!forum/wazuh
+-->
+
+<ossec_config>
+  <client>
+    <server>
+      <address>MANAGER_IP</address>
+      <port>1514</port>
+      <protocol>tcp</protocol>
+    </server>
+    <config-profile>debian, debian10</config-profile>
+    <notify_time>10</notify_time>
+    <time-reconnect>60</time-reconnect>
+    <auto_restart>yes</auto_restart>
+    <crypto_method>aes</crypto_method>
+  </client>
+
+  <client_buffer>
+    <!-- Agent buffer options -->
+    <disabled>no</disabled>
+    <queue_size>5000</queue_size>
+    <events_per_second>500</events_per_second>
+  </client_buffer>
+
+  <!-- Policy monitoring -->
+  <rootcheck>
+    <disabled>no</disabled>
```

Comment:
Why isn't it disabled?

Reply:
The modules are not disabled, since the enviroment enviroment is generic. Also, the test that uses this environment at the moment does not check logs, so  having modules enabled does not affect the test.